front/lib/actions/server.ts

```diff
@@ -92,7 +93,30 @@ export async function runActionStreamed(
     return new Err(res.error);
   }
 
+  // Don't await the dustRunid promise before you are done iterating on the eventStream,
```

Comment:
We have craeted a deadlock here, so a little comment to help future us not waste time on it.

Reply:
Tricky indeed should we stop returning that promise outside of this function?